okr_app/tests.py

```diff
@@ -1,4 +1,100 @@
-# flake8: noqa
 from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import SimpleUploadedFile
+from okr_app.models import OKR, OKRFile
 
-# Create your tests here.
+class TaskDetailAPITestCase(TestCase):
+    #Test preparation, create users, okr and okrfiles instances
+    def setUp(self):
+        self.client = APIClient()
+        fileSample = SimpleUploadedFile("file.pdf", b"file_content", content_type="application/pdf")
+        self.user = User.objects.create(username='okrowner')
+        self.user.set_password('12345')
+        self.user.save()
+        self.userGuest = User.objects.create(username='guest')
+        self.userGuest.set_password('12345')
+        self.userGuest.save()
+        self.okr = OKR(issuer=self.user, quarter="1", year="2021", content="test content")
+        self.okr.save()
+        self.okr_file = OKRFile(issuer=self.user, file=fileSample, okr=self.okr)
+        self.okr_file.save()
+
+    #Can OKR issuer get list of all his/her okr files?
```

Comment:
using affirmative to describe test case, do not use interrogative, same for the others

Reply:
put description in side the function with """ """
```
e.g:
def testcase(self):
    """testcase
    """
```